spec/views/cases/_case_request_html_slim_spec.rb

```diff
@@ -36,20 +36,8 @@
   let(:empty_message) { "" }
```

Comment:
we don't need this to be a let (also, the preferred style (which I detest) is single quotes unless you're interpolating) - we can delete this